target/linux/rockchip/image/armv8.mk

```diff
@@ -47,6 +47,14 @@ define Device/friendlyarm_nanopi-r4s
 endef
 TARGET_DEVICES += friendlyarm_nanopi-r4s
 
+define Device/friendlyarm_nanopi-r5c
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi R5C
+  SOC := rk3568
+  DEVICE_PACKAGES := kmod-r8169 kmod-rtw88 rtl8822ce-firmware wpad-basic-mbedtls
```

Comment:
should be kmod-rtw88-8822ce instead of kmod-rtw88?

Reply:
Thanks, updated.